Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -0,0 +1,100 @@
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+      <startup>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7"/>
+      </startup>
+      <runtime>
+        <!-- AppContextSwitchOverrides value attribute is in the form of 'key1=true|false;key2=true|false  -->
+        <AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=false" />
+      </runtime>
+    </configuration>
+
+If an application targets version 4.7.1 or above and has preserve the legacy accessibility behavior, please opt in into the use of legacy accessibility features explicitly by setting this AppContext switch value to *true*. 
+
+__Support for UI Automation__
+- Added [UIA](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-overview) [Expand/Collapse pattern](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern) to `T:System.Windows.Forms.ToolStripSplitButton`
+- Added UIA [ControlType](https://msdn.microsoft.com/en-us/library/windows/desktop/ee671197(v=vs.85).aspx) property value "MenuItem" to `T:System.Windows.Forms.ToolStripMenuItem`
+- Added UIA [Name](https://msdn.microsoft.com/en-us/library/dd757479(v=vs.85).aspx) property and Expand/Collapse pattern to `T:System.Windows.Forms.ToolStripItem`
+- Added [AccessibleEvents](https://msdn.microsoft.com/en-us/library/system.windows.forms.accessibleevents(v=vs.110).aspx) indicating StateChange and NameChange when drop down is expanded or collapsed to `T:System.Windows.Forms.ToolStripDropDownItem`
+- Added UIA ControlType property value "Button" to `T:System.Windows.Forms.ToolStripDropDownButton`
+- Added UIA [Toggle pattern](https://msdn.microsoft.com/en-us/library/system.windows.automation.togglepattern(v=vs.110).aspx) to `T:System.Windows.Forms.DataGridViewCheckBoxCell`
+ 
+__Improvements to PropertyBrowser control__
+- Added UIA Expand/Collapse pattern, state and name change notifications and ControlType property with value "Button" to the "Details" button in the error dialog that is displayed when the user enters incorect value in the `T:System.Windows.Forms.PropertyGrid` control 
+- Changed [AccessibleRole](https://msdn.microsoft.com/en-us/library/system.windows.forms.control.accessiblerole(v=vs.110).aspx) of rows in PropertyGrid control from "Row" to "Cell", which maps to UIA ControlType "DataItem", in order to support appropriate keyboard shortcuts and Narrator announcements and UIA ControlType "DataItem"
+- Improved keyboard navigation between the grid and the ToolBar above it. Pressing "Shift-Tab" now selects the first ToolBar button, instead of the whole ToolBar
+
+__Support for High Contrast themes__
+
+__Improved keyboard navigation__
+
+### Affected APIs
+Added new public override methods or properties
```

Comment:
Remove this. This is ignored when we parse the documents.

Reply:
done